api/routes.py

```diff
@@ -92,7 +101,7 @@ def get_fund(fund_id):
     if use_short_name:
         fund = get_fund_by_short_name(fund_id)
     else:
-        fund = get_fund_by_id(fund_id)
+        fund = get_fund_by_id(fund_id) if is_valid_uuid(fund_id) else None
```

Comment:
Should we add this check here as well? https://github.com/communitiesuk/funding-service-design-fund-store/pull/334/files#diff-e8bf8155ecc27651ffc56921d9f536dc9717374e8315f9f1cf384ac08513f621R120

Reply:
On `round_id`

Reply:
Added check for round_id and event_id